script.js

```diff
@@ -0,0 +1,50 @@
+//name request//
+var person = prompt("Please enter your name", "Kevin Hart");
+
+if (person != null) {
+    console.log("Hey! " + person + "! Are you ready for a ride along with Ice Cube?");
+
+}
+
+var text;
+var favSong = prompt("What's your favorite Ice Cube Song");
+switch(favSong) {
```

Comment:
One thing i would mention, it I would make the prompt something a little bit more manageble for users. Give them just one letter to enter that represents the song. Long strings like the one in your case statements are much more error prone